Makefile

```diff
@@ -0,0 +1,97 @@
+GOOS:=linux
```

Comment:
The `artifacts` target covers that, but I think we're open to only doing gox, if that's preferred.